dist/leaflet.css

```diff
@@ -330,6 +330,25 @@ svg.leaflet-image-layer.leaflet-interactive path {
 	border-bottom-right-radius: 2px;
 	}
 
+
+/* Dark Mode */
+.leaflet-darkmode .leaflet-bar a,
+.leaflet-darkmode .leaflet-bar a:hover{
+	background-color: #333;
+	border-bottom: 1px solid #ddd;
+	color: #fff;
+	}
+
+.leaflet-darkmode .leaflet-bar a:hover {
+	background-color: #444;

```

Comment:
Probably add a selector for `:focus` per https://github.com/Leaflet/Leaflet/pull/7274.